src/components/transactions/TransactionRow.vue

```diff
@@ -393,6 +402,27 @@ export default {
           console.error(err)
         })
     },
+    async claimClick() {
+      const data = this
```

Comment:
maybe I'd extract `data.sharedState` to a `let sharedState = data.sharedState` since we're using it multiple times (and writing to it) below

Reply:
ok

Reply:
done